lib/manageiq/messaging.rb

```diff
@@ -1,4 +1,5 @@
 require 'active_support/core_ext/module/delegation'
+require 'active_support/core_ext/hash'
```

Comment:
is it possible to get rid of `active_support` all together?

The first usage of `delegation` could have been done by `Forwardable`
Not sure why we're adding `hash`.